openlibrary/templates/books/custom_carousel.html

```diff
@@ -0,0 +1,38 @@
+$def with(books=None, title="", url="", key="", min_books=6, cta="Borrow", test=False)
+
+$if test or (books and len(books) >= min_books):
```

Comment:
Is this at the right indentation level, given the `def` above?

Reply:
yup